### PR TITLE
docs(vue): Point to new Vue docs, fix scaffold command

### PR DESCRIPTION
**Pull request by bsmth**
The [Vue todo repo](https://github.com/mdn/todo-vue) has some fixes to get build & deploys running again, importantly bumping to V3. Most of the updates are to the scaffold CLI command which defaults to Vue 3 & pointing to new docs.

The guide doesn't use any features that have [breaking changes](https://v3-migration.vuejs.org/breaking-changes/).

Other changes are Prettier formatting which I've tried to reduce to a minimum.

### CODE REVIEW

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_methods_events_models/index.md [hunk 1]
```diff
@@ -89,15 +89,14 @@ We now have an app that displays a list of to-do items. However, we can't update
...
+   import ToDoForm from "./components/ToDoForm";
    ```
 
 5. You also need to register the new component in your `App` component — update the `components` property of the component object so that it looks like this:
 
    ```js
    components: {
-     ToDoItem,
-     ToDoForm
+     ToDoItem, ToDoForm;
```
Comment (chrisdavidmills) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_methods_events_models/index.md:99]: Seems a little unusual to put semi-colons on the end of object properties?

**Review by chrisdavidmills — COMMENTED** [review]
Hi @bsmth ! Looking pretty good to me; only one minor comment.

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_methods_events_models/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_methods_events_models/index.md:99]: This was applied by Prettier, I'll revert 👍🏻

Comment (bsmth) [reply files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_methods_events_models/index.md]: Done in 480f178919a42f4ea56e9e3becca4d8e54fa0f5e

**Comment by bsmth** [comment]
Thanks @chrisdavidmills, going to merge shortly 👍🏻

**Review by Rumyra — APPROVED** [review]

**Comment by bsmth** [comment]
Thanks @Rumyra 🙌🏻

**Pull request merged**